src/server/h1.rs

```diff
@@ -1205,7 +1205,7 @@ mod tests {
         // type in chunked
         let mut buf = Buffer::new(
             "GET /test HTTP/1.1\r\n\
-             transfer-encoding: chnked\r\n\r\n");
+             transfer-encoding: chunked\r\n\r\n");
```

Comment:
This is actually test for broken transfer encoding

Reply:
oops